src/Trixi.jl

```diff
@@ -39,6 +39,7 @@ import SciMLBase: get_du, get_tmp_cache, u_modified!,
                   get_proposed_dt, set_proposed_dt!,
                   terminate!, remake, add_tstop!, has_tstop, first_tstop
 
+using Downloads: download
```

Comment:
If you use something like the suggestion in auxiliary.jl:

```suggestion
using Downloads: Downloads
```